src/components/CollectionProvider/CollectionProvider.container.ts

```diff
@@ -0,0 +1,29 @@
+import { connect } from 'react-redux'
+import { isLoadingType } from 'decentraland-dapps/dist/modules/loading/selectors'
+import { isConnecting } from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { RootState } from 'modules/common/types'
+import { getCollectionId } from 'modules/location/selectors'
+import { isLoggingIn } from 'modules/identity/selectors'
+import { getLoading, getCollections, getCollectionItems } from 'modules/collection/selectors'
+import { FETCH_COLLECTION_REQUEST, fetchCollectionRequest } from 'modules/collection/actions'
+import { MapStateProps, MapDispatchProps, MapDispatch, OwnProps } from './CollectionProvider.types'
+import CollectionProvider from './CollectionProvider'
+
+const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
+  const id = ownProps.id || getCollectionId(state)
+  const collections = getCollections(state)
```

Comment:
doesn't this only return collections owned by the logged in user? If i'm logged in as a different user as the owner, wouldn't this always fail to find the collection i'm looking for?

Reply:
Yes! I was testing with an specific case and forgot